tests/settings.py

```diff
@@ -120,3 +120,4 @@
 
 STATIC_ROOT = os.path.join(BASE_DIR, "admin_interface/public/static/")
 STATIC_URL = "/static/"
+LOCAL_FILE_DIR = os.path.join(BASE_DIR, "admin_interface/public/")
```

Comment:
I think this name is too generic (doesn’t indicate it’s for django-admin-interface) and at the same time too vague! (local file dir? what is a file dir, what is a non-local file dir)

So: I think this could be named something like `ADMIN_INTERFACE_STATIC_LOGO_PATH`, with an example value of `theme-logos/logo.png`. Then let django use the normal static file system to resolve the actual full path to `project-dir/some-app/theme-logos/logo.png`.

Reply:
Maybe change the settings name with `ADMIN_INTERFACE_STATIC_LOGO_PATH` is better

Reply:
Considering that this is a setting for specifying the static logo directory, I would name it `ADMIN_INTERFACE_STATIC_LOGO_DIR`.